inc/saiswitch.h

```diff
@@ -3048,6 +3048,42 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_MAX_ICMP_ECHO_SESSION,
 
+    /**
+     * @brief List of TCs eligible for trimming
+     *
+     * @type sai_u8_list_t
+     * @flags CREATE_AND_SET
+     * @default empty
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_ENABLE_TC_LIST,
```

Comment:
if we set the drop action as Trim on buffer profile (used for Queue) or directly on Queue why we need this attribute ?

Reply:
To be removed.